### DISCUSSION TITLE
Why SIM Swap has introduced wild-card scope?

### DISCUSSION THREAD

**Discussion by ToshiWakayama-KDDI** [camaraproject/SimSwap#178] [Q&A]
Dear Community,

In the SIM Swap API YAML specification, I found the wild-card scope 'sim-swap' specified, in addition to 'sim-swap:retrieve-date' and 'sim-swap:check', and also found that this was done by PR #103 'add API-Name aka wild-card scope' (https://github.com/camaraproject/SimSwap/pull/103) .

I understand in CAMARA wild-card scope is possible and allowed, but I cannot find any other APIs adopting wild-card scope than SIM Swap.  I don't know why SIM Swap only introduced wild-card scope, and I cannot fine the reason in GitHub.  Could you advise me why SIM Swap has introduced the wild-card scope?

Many thanks,

**Comment by fernandopradocabrillo** [comment]
The argument was that it makes sense for the client to be able to consume both endpoints using the same scope, as the functionality is related.
